k2/csrc/pytorch_context.h

```diff
@@ -50,8 +59,14 @@ class PytorchCpuContext : public Context {
     return p;
   }
 
-  void Deallocate(void *data, void * /*deleter_context*/) override {
-    allocator_->raw_deallocate(data);
+  void Deallocate(void *data, void *deleter_context) override {
+    if (deleter_context) {
```

Comment:
use `deleter_context != null_ptr`? as we talking about the style before (for any type except bool,  we always use `!=` or `==` to check its value)

Reply:
fxied.